index.html

```diff
@@ -6,6 +6,44 @@
   <body>
       <h1 id="title">Survey Form</h1>
       <form id="survey-form">
+      <label 
+        for="Sure">
+        <input id="sure"
```

Comment:
On elements like this where we are separating the attributes with new line, it should go: tagname, new line, attribute, attribute, ....

```
      <input 
        id="sure"
        type="checkbox" 
        value="sure"
        name="recommend"
        >
```

Not

```
      <input id="sure"
        type="checkbox" 
        value="sure"
        name="recommend"
        >
```